priv/static/swagger_v1.yaml

```diff
@@ -1480,6 +1576,476 @@ info:
+    get:
+      deprecated: false
+      description: Get all AEX9 balances for an account at certain block.
+      operationId: get_aex9_account_hash_balances
+      parameters:
+        - description: Hash of a key or micro block
+          in: path
+          name: blockhash
+          required: true
+          type: string
+          x-example: mh_U1Hm7NTSwo37Lpf2ZmTZjS1j67ptqXqzWYBjQJPdFQjA939cm      
+        - description: Account id
+          in: path
+          name: account_id
+          required: true
+          type: string
+          x-example: ak_152krtQsp2T3ApcFv7B21FFXkjXnsps3VBfhJeyNF1tHh6mB2
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Returns all the balances of an account on key or micro block 
+          schema: 
+            type: array
+            items: 
+              $ref: "#/definitions/Aex9BalanceResponse"
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+      summary: ''
+      tags:
+        - Middleware     
+  /aex9/balances/hash/{:blockhash}/{:contract_id}:
+    get:
+      deprecated: false
+      description: Get AEX9 balances on a contract at certain block.
+      operationId: get_aex9_contract_hash_balances
+      parameters:
+        - description: Hash of a key or micro block
+          in: path
+          name: blockhash
+          required: true
+          type: string
+          x-example: mh_U1Hm7NTSwo37Lpf2ZmTZjS1j67ptqXqzWYBjQJPdFQjA939cm      
+        - description: Contract id
+          in: path
+          name: contract_id
+          required: true
+          type: string
+          x-example: ct_uGk1rkSdccPKXLzS259vdrJGTWAY9sfgVYspv6QYomxvWZWBM
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Returns balances on a contract at certain block
+          schema: 
+            $ref: "#/definitions/Aex9ContractBalancesResponse"
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+      summary: ''
+      tags:
+        - Middleware
+  /aex9/balances/{:contract_id}:
+    get:
+      deprecated: false
+      description: Get AEX9 balances on a contract.
+      operationId: get_aex9_contract_balances
+      parameters:
+        - description: Contract id
+          in: path
+          name: contract_id
+          required: true
+          type: string
+          x-example: ct_uGk1rkSdccPKXLzS259vdrJGTWAY9sfgVYspv6QYomxvWZWBM
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Returns balances on a contract
+          schema: 
+            $ref: "#/definitions/Aex9ContractBalancesResponse"
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+      summary: ''
+      tags:
+        - Middleware
+  /aex9/balances/gen/{:range}/{:contract_id}:
```

Comment:
why is this PR editing v1 as well? it appears the same errors from before as present on v1 as well